File: Lab Practice 1/Marks.c
#include <stdio.h>
int main() {
    float marks[5], total, average, percentage;

    printf("Enter marks for Subject 1: ");
    scanf("%f", &marks[0]);

    printf("Enter marks for Subject 2: ");
    scanf("%f", &marks[1]);

    printf("Enter marks for Subject 3: ");
    scanf("%f", &marks[2]);

    printf("Enter marks for Subject 4: ");
    scanf("%f", &marks[3]);

    printf("Enter marks for Subject 5: ");
    scanf("%f", &marks[4]);

    total = marks[0] + marks[1] + marks[2] + marks[3] + marks[4];
    average = total / 5;
    percentage = (total / (5 * 100)) * 100;

    printf("Total marks: %.2f\n", total);
    printf("Average marks: %.2f\n", average);
    printf("Percentage: %.2f%%\n", percentage);

    return 0;
}

File: Lab Practice-1/Marks.java
import java.util.Scanner; 
public class StudentResultCalculator { 
public static void main(String[] args) { 
Scanner scanner = new Scanner(System.in); 

System.out.println("Enter marks for each subject:"); 

        double subject1, subject2, subject3, subject4, subject5; 

        System.out.print("Subject 1: "); 
        subject1 = scanner.nextDouble(); 
 
        System.out.print("Subject 2: "); 
        subject2 = scanner.nextDouble(); 
 
        System.out.print("Subject 3: "); 
        subject3 = scanner.nextDouble(); 
 
        System.out.print("Subject 4: "); 
        subject4 = scanner.nextDouble(); 
 
        System.out.print("Subject 5: "); 
        subject5 = scanner.nextDouble(); 

        double total = subject1 + subject2 + subject3 + subject4 + subject5; 
        double average = total / 5; 
        double percentage = (total / 500) * 100; 
 
        System.out.printf("Total marks: %.2f%n", total); 
        System.out.printf("Average marks: %.2f%n", average); 
        System.out.printf("Percentage: %.2f%%%n", percentage); 
 
        scanner.close(); 
    } 
} 
